execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -2959,3 +2959,51 @@ fn should_validate_genesis_delegators_bond_amount() {
 
     builder.run_genesis(&run_genesis_request);
 }
+
+fn check_validator_slots_for_accounts(accounts: usize) {
+    let accounts = {
+        let range = 1..=accounts;
+        let (capacity, _) = range.size_hint();
```

Comment:
Isn't `capacity == accounts` now? Would it be easier to just use `accounts`?
Alternatively, don't make a mutable vector at all:
```rust
let accounts: Vec<_> = (1..=accounts).map(|i| {
    let count = U256::from(i);
    let mut secret_key_bytes = [0; 32];
    …
}).collect();
```

Reply:
Oh, right, that `size_hint` was a bit of an overkill 🤦 I just used `accounts` value. Thanks!